t/lib/CorpusTest.pm

```diff
@@ -4,11 +4,13 @@ use warnings;
 use Test::More 0.96;
 use Test::Deep qw/!blessed/;
 
+use JSON::PP ();
+use JSON::XS ();
```

Comment:
I've actually now switched it to `Cpanel::JSON::XS`. The `XS`/`PP` usage is because we require `PP` for the ability to created an `IxHash`, but we need `XS` for proper svtype detection. The leftover usage of `MaybeXS` was from earlier, and I'm now using the `XS` variant explicitly.

Reply:
Can you talk this over with @shadowcat-mst please?  Keeping BSON pure-Perl capable -- even in testing -- is pretty important; part of its reason for being, even.  If there's no way to avoid XS for testing, then it needs to be conditional.